elasticdl/python/master/pod_manager.py

```diff
@@ -188,10 +201,11 @@ def __init__(
         **kwargs
     ):
         self._num_workers = num_workers
+        self._chief_worker_index = chief_worker_index
         self._worker_resource_request = worker_resource_request
         self._worker_resource_limit = worker_resource_limit
         self._worker_pod_priority = _parse_worker_pod_priority(
-            self._num_workers, worker_pod_priority
+            self._num_workers, worker_pod_priority, self._chief_worker_index
```

Comment:
Can we set the worker-0 as the chief worker. Because the worker has high priority is the `worker_pod_priority` is no `low`. Then, we don't need any codes to set the chief worker.

Reply:
For estimator, the last worker is the chief worker for evaluation.